operator/pkg/ciliumendpointslice/cescache.go

```diff
@@ -50,6 +55,11 @@ func NewSecIDs() *SecIDs {
 type CESCache struct {
 	mutex lock.RWMutex
 
+	// cesData is used to map CiliumEndpointSlice name to all CiliumEndpoints names it contains
+	// and the namespace associated with it
+	cesData map[CESName]*CESData
+	// nsData is used to map namespaces to all CiliumEndpointSlices in them
+	nsData map[string]sets.Set[CESName]
```

Comment:
This does not work correctly if two CESes have the same name in different namespaces. Today CESes are cluster scoped, so that is effectively impossible, but it may be something to keep in mind if that were to change in the future.